_pages/algorithm-development/your-first-algo.md

```diff
@@ -13,224 +13,215 @@ redirect_from:
 Depending on your data source, you'll either want to use <a href="http://docs.algorithmia.com/#data-api-specification">Algorithmia's Data API</a> to connect to your data within your algorithm, or write a wrapper to connect to your database. 
 
-To learn about the various data sources, make sure to visit <a href="{{site.baseurl}}/data">Connecting Your Data</a>, or check out our reference guide on the available methods using <a href="http://docs.algorithmia.com/#data-api-specification">Algorithmia's Data API</a>.
+To learn about the various data sources that Algorithmia supports natively, visit <a href="{{site.baseurl}}/data">Connecting Your Data</a> or check out our reference guide on the available methods using <a href="http://docs.algorithmia.com/#data-api-specification">Algorithmia's Data API</a>.
 
-Note: if you are authoring an algorithm and using the Data API, avoid using the ‘.my’ pseudonym in the source code. When the algorithm is executed, ‘.my’ will be interpreted as the user name of the user who called the algorithm, rather than the author’s user name.
+If you are authoring an algorithm and using the Data API, avoid using the `.my` pseudonym in the source code. When the algorithm is executed, `.my` will be interpreted as the user name of the user who called the algorithm, rather than the author’s user name.
 {: .notice-warning}
 
 ## Calling Your Algorithm
-To call your algorithm during testing you can use the hash of that algorithm to call it locally using the language client of your choice. Or if you are testing it within the Algorithmia Web IDE you can use the console for running your algorithm during development. 
+To call your algorithm during development and testing, you can use the commit hash to call it locally using the language client of your choice. If you are developing within the Algorithmia Web IDE, you can use the built-in console for testing. 
 
-To call a private version of an algorithm you own, you must use a fully specified semantic version or a version hash or you will get an error. For more information you can get details in the <a href="https://algorithmia.com/developers/api/#api-specification">API Docs</a>
+To call a private version of an algorithm you own, you must use a fully specified semantic version or a version hash or you will get an error. For more information see the details in the <a href="https://algorithmia.com/developers/api/#api-specification">API Docs</a>
 {: .notice-warning}
 
-## Publish Your Algorithm
-Last is publishing your algorithm. The best part of hosting your model on Algorithmia is that users can access it via an API that takes only a few lines of code to use! Here is what you can set when publishing your algorithm:
+## Publishing Your Algorithm
+The last step is publishing your algorithm. The best part of hosting your code on Algorithmia is that users can access it via an API that takes only a few lines of code to use! Here is what you can set when publishing your algorithm:
```

Comment:
Lets make this a little more enterprise friendly - something like "The last step is publishing your algorithm. Deploying your code to your Algorithmia cluster will provide you with an endpoint to share that will scale automatically no matter how often the API is called" - this isn't great, but something along those lines

Reply:
how about: "The last step is publishing your algorithm. Once your code is deployed on your Algorithmia cluster, you will have a shareable endpoint that will scale automatically to handle request load, no matter how often the API is called."

Reply:
perfect use that!